clang/include/clang/Basic/DiagnosticASTKinds.td

```diff
@@ -389,6 +389,12 @@ def note_constexpr_assumption_failed : Note<
   "assumption evaluated to false">;
 def err_experimental_clang_interp_failed : Error<
   "the experimental clang interpreter failed to evaluate an expression">;
+def note_unaligned_atomic_pointer_op : Note<
+  "atomic pointer operation with argument %0 not aligned to size of pointee type (sizeof %1 is %2)">;
+def note_constexpr_atomic_ops_only_in_cpp : Note<
```

Comment:
This note is not emitted anywhere, but also, I don't think a note's the correct way to go because that will fail at constant expression time but `__has_constexpr_builtin` will still return true.